project/plugins.sbt

```diff
@@ -20,6 +20,7 @@ addSbtPlugin("com.lightbend" % "sbt-whitesource" % "0.1.13")
 addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.0.0") // for maintenance of copyright file header
 addSbtPlugin("com.hpe.sbt" % "sbt-pull-request-validator" % "1.0.0")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.0-M3")
```

Comment:
There is a M5 out (do we really want to depend on a milestone?)

Reply:
My reasoning is a one-off, since this is just a post-build nightly task in Jenkins and the release is not far off that it is ok; I'd upgrade asap. Otherwise no, I'd not want a milestone. Will do m5.

Reply:
@johanandren and the primary reason for 1.6 is `coverageAggregate`

Reply:
Alright, thanks for clarifying!

Reply:
But I should have clarified that up front :-)